spec/models/manageiq/providers/redfish/inventory/collector/physical_infra_manager_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe ManageIQ::Providers::Redfish::Inventory::Collector::PhysicalInfraManager do
+  subject(:collector) { described_class.new(nil, nil) }
+
+  context "#get_server_location" do
+    it "returns only id on missing chassis" do
+      server_data = {
+        "@odata.id" => "sid",
+        "Links"     => { "Chassis" => [] }
```

Comment:
Can the Chassis be `nil`?

Reply:
No. Redfish collections are always present, but can be empty.